src/index.html

```diff
@@ -1,16 +1,56 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="wrapper">
+      <div class="radius">
+        <form class="form"
+          autofocus
+          action="value"
+          method="get"
+          data-qa="big"
+        >
```

Comment:
```suggestion
        <form 
          class="form"
          autofocus
          action="value"
          method="get"
          data-qa="big"
        >
```
Please, fix the code style everywhere. In this situation, you should apply this rule. But please, read the checklist carefully and refactor your code.
https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md
![image](https://user-images.githubusercontent.com/102293862/232562937-3bf14b97-22bc-41e6-8224-9e21b08135b8.png)

Reply:
Я помітив, клас перенести на наступну строку?